### ISSUE TITLE
t/02-notify.t failures

### ISSUE THREAD

**Issue by eserte** [gnustavo/Git-Hooks#44]
t/02-notify.t fails on some of my smokers:
```
#   Failed test 'default subject (no mailbox)'
#   at t/02-notify.t line 45.

#   Failed test 'subject replace all placeholders (no mailbox)'
#   at t/02-notify.t line 45.

#   Failed test 'from header (no mailbox)'
#   at t/02-notify.t line 45.

#   Failed test 'to header (no mailbox)'
#   at t/02-notify.t line 45.

#   Failed test 'preamble (no mailbox)'
#   at t/02-notify.t line 45.

#   Failed test 'commit-url (no mailbox)'
#   at t/02-notify.t line 45.

#   Failed test 'do notify if match pathspec (no mailbox)'
#   at t/02-notify.t line 45.

#   Failed test 'html (no mailbox)'
#   at t/02-notify.t line 45.
# Looks like you failed 8 tests of 9.
t/02-notify.t ............ 
Dubious, test returned 8 (wstat 2048, 0x800)
Failed 8/9 subtests 
```
It seems that this happens on Debian wheezy+jessie and FreeBSD 9.2 systems. The test suite passes on Debian stretch, Ubuntu 16.04 and FreeBSD >= 10.

**Comment by eserte** [comment]
Maybe the git version is relevant:
```
wheezy - git version 1.7.10.4
jessie - git version 2.1.4
FreeBSD 9.2-RELEASE-p4 - git version 2.4.0
xenial - git version 2.7.4
FreeBSD 10.3-RELEASE-p11 - git version 2.11.0
stretch - git version 2.11.0
FreeBSD 11.1-RELEASE-p4 - git version 2.14.1
FreeBSD 12.0-CURRENT - git version 2.15.1
FreeBSD 11.1-STABLE - git version 2.15.1
```
So it works starting from git version 2.7.4.

**Comment by gnustavo** [comment]
Thank you! I'll take a look into it.

**Comment by gnustavo** [comment]
Hi. I just released version 2.2.1 which should fix this.

Thanks!
